### PR TITLE
updated get_feed() function in Hashnode Scraper

### PR DESCRIPTION
**Pull request by Mihan786Chistie**
## Description

> Improved the get_feed() function as per the new design of the hashnode website

## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [x] Have you adhered to the repository's defined coding convention rules?
- [x] Have you updated the 'documentation.md' file with the method/function documentation?
- [x] Have you sent a message along with the result or response?
- [x] Have you used the try-catch technique?
- [x] Has the method/class been added to the documentation (md file)?

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

File: src/scrape_up/hashnode/hashnode.py [hunk 1]
```diff
@@ -44,48 +43,76 @@ def get_feed(self):
 
             blogs_data = {"blogs": []}
 
-            blogs = soup.find_all("div", class_="css-1s8wn94")
+            blogs = soup.find_all(
+                "div",
```
Comment (nikhil25803) [inline src/scrape_up/hashnode/hashnode.py:47]: This isn't working @Mihan786Chistie | `blogs` is returning an empty list.

Please check once

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/hashnode/hashnode.py (hunk 1 above)
Comment (Mihan786Chistie) [reply src/scrape_up/hashnode/hashnode.py]: Check it now @nikhil25803

Comment (nikhil25803) [reply src/scrape_up/hashnode/hashnode.py]: Still returning empty array @Mihan786Chistie

**Comment by Mihan786Chistie** [comment]
Are you sure? I am getting the output to the new changes:
hashnode_updated

**Comment by Mihan786Chistie** [comment]
@nikhil25803 please review it

**Review by nikhil25803 — APPROVED** [review]

**Comment by Mihan786Chistie** [comment]
@nikhil25803 please add the level tag

**Pull request merged**